### PR TITLE
Bug in NSG rule for health check? (incorrect direction, or incorrect description?)

### LINKED ISSUE #820: Bug in NSG rule for health check? (incorrect direction, or incorrect description?) v5.0.0-RC4
Rule `module.oke.module.network.oci_core_network_security_group_security_rule.oke["Allow TCP ingress to workers for health check from OKE control plane"]` appears to have either an incorrect description or incorrect direction. Please advise.

```  
# module.oke.module.network.oci_core_network_security_group_security_rule.oke["Allow TCP ingress to workers for health check from OKE control plane"] will be created
  + resource "oci_core_network_security_group_security_rule" "oke" {
      + description               = "Allow TCP ingress to workers for health check from OKE control plane"
      + destination               = "ocid1.networksecuritygroup.oc1.phx.xxx"
      + destination_type          = "NETWORK_SECURITY_GROUP"
      + direction                 = "EGRESS"
      + id                        = (known after apply)
      + is_valid                  = (known after apply)
      + network_security_group_id = "ocid1.networksecuritygroup.oc1.phx.xxx"
      + protocol                  = "6"
      + source_type               = (known after apply)
      + stateless                 = false
      + time_created              = (known after apply)

      + tcp_options {
          + destination_port_range {
              + max = 10250
              + min = 10250
            }
        }
    }
```

Thank you.

### PR DESCRIPTION
**Pull request by syedthameem85**
Resolves #820

### CODE REVIEW

File: modules/network/nsg-workers.tf [hunk 1]
```diff
@@ -33,7 +33,7 @@ locals {
       "Allow TCP egress from workers to OKE control plane" : {
         protocol = local.tcp_protocol, port = local.oke_port, destination = local.control_plane_nsg_id, destination_type = local.rule_type_nsg,
       },
-      "Allow TCP ingress to workers for health check from OKE control plane" : {
+      "Allow TCP egress to workers for health check from OKE control plane" : {
```
Comment (hyder) [inline modules/network/nsg-workers.tf:36]: This should be Allow TCP egress to OKE control plane from workers for health check

**Review by hyder — CHANGES_REQUESTED** [review]
please review the recommended changes

**Comment by syedthameem85** [comment]
@hyder -The change has already been done and can be merged.

**Comment by hyder** [comment]
> @hyder -The change has already been done and can be merged.

your description is still incorrect.

**Comment by syedthameem85** [comment]
@hyder  - " Allow TCP egress to OKE control plane from workers for health check"  Is this the correct description ?

**Review by hyder — APPROVED** [review]
Approved

**Pull request merged**